### PR TITLE
Feat/pyinstaller

### PR DESCRIPTION
**Pull request by 0xArdi**
This PR adds support for release teh trader aea as a binary.

### CODE REVIEW

File: pyinstaller/trader_bin.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# ------------------------------------------------------------------------------
+
+"""Script for building the AEA responsible for running the trader."""
+import os
+import sys
+from pathlib import Path
+import aea.configurations.validation as validation_module
+
+# patch for the _CUR_DIR value
+validation_module._CUR_DIR = Path(sys._MEIPASS) / validation_module._CUR_DIR
```
Comment (jmoreira-valory) [inline pyinstaller/trader_bin.py:27]: Is this only valid for Mac ?

Comment (Adamantios) [reply pyinstaller/trader_bin.py:27]: Could you update the comment to explain why we need this patch?

Comment (0xArdi) [reply pyinstaller/trader_bin.py:27]: no, its for all, just a pyinstaller thing

Comment (0xArdi) [reply pyinstaller/trader_bin.py:28]: added a comment

**Review by Adamantios — APPROVED** [review]

**Pull request merged**